### PR TITLE
kvrpcpb: add isolation level

### PR DESCRIPTION
**Pull request by nolouch**
PTAL  @disksing @zhangjinpeng1987 @siddontang

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by zhangjinpeng1987** [comment]
@disksing PTAL

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -30,6 +30,11 @@ enum CommandPri {
     High = 2;
 }
 
+enum IsolationLevel {
+    SI = 0;
+    RU = 1;
```
Comment (disksing) [inline proto/kvrpcpb.proto]: Please add comments for them: SI=snapshot isolation, RU=read uncommitted。

**Comment by disksing** [comment]
LGTM.

**Pull request merged**